### PR TITLE
Reorganize getting started

### PR DESCRIPTION
**Pull request by Obliviour**
getting started is now organized into three docs, each describing a tool we have. Readme.md links to these three docs.

Deleted other xpk files that are no longer needed.

### CODE REVIEW

File: getting_started/Run_MaxText_via_xpk.md [hunk 1]
```diff
@@ -84,7 +84,9 @@ after which log out and log back in to the machine.
     gcloud config set project $PROJECT_ID
     bash docker_upload_runner.sh CLOUD_IMAGE_NAME=${USER}_runner
```
Comment (Obliviour) [inline getting_started/Run_MaxText_via_xpk.md:85]: @rwitten I was curious if you had a good way that we could avoid the user needing to upload the runner at all?

Do we build the dependency image already and then they could just use that as their base docker image? Or is this worth (could have mismatch to local maxtext)

**Review by rwitten — CHANGES_REQUESTED** [review]

File: getting_started/Run_MaxText_via_xpk.md (hunk 1 above)
Comment (Obliviour) [reply getting_started/Run_MaxText_via_xpk.md:85]: This is a good idea. XPK doesn't support this yet so I am thinking of adding this feature as a precursor to the PR here

File: README.md [hunk 2]
```diff
@@ -26,14 +26,20 @@ MaxText aims to be a launching off point for ambitious LLM projects both in rese
 # Table of Contents
 
 * [Getting Started](#getting-started)
-* [Runtime Performance Results](#runtime-performance-results)
```
[the remark this answers is not in the document]
Comment (Obliviour) [reply README.md:29]: Accidently! Thanks for the find!

File: getting_started/Run_MaxText_via_xpk.md (hunk 1 above)
Comment (Obliviour) [reply getting_started/Run_MaxText_via_xpk.md:85]: Actually I am incorrect. XPK does support this and I somehow didn't think it did.

File: README.md [hunk 3]
```diff
@@ -68,146 +69,12 @@ python3 MaxText/decode.py MaxText/configs/base.yml run_name=$YOUR_JOB_NAME
...
-    python3 multihost_job.py --NUM_SLICES=$NODE_COUNT --RUN_NAME=$RUN_NAME --BUCKET_NAME=$BUCKET_NAME --CQR_EXTRA_ARGS="--reserved" --COMMAND="bash setup.sh && python3 MaxText/train.py MaxText/configs/base.yml run_name=$RUN_NAME"
-    ```
-
-    We tell `multihost_job` to target the `reserved` pool by  by including `--reserved` as extra arguments to the CQR request, but you may instead target the `on-demand` pool by removing the `--CQR_EXTRA_ARGS` flag (on-demand is default), or the pre-emptible pool with `--CQR_EXTRA_ARGS="--best-effort"`, which may be necessary if your reservation is full.
-
-5. View the job's logs in cloud logging. 
-
-    The link to your job's cloud logging is printed at the end of `multihost_job` output. Additionally logs are saved to GCS when your job finishes, and this bucket's URL is also printed by `multihost_job`.
+## Getting Starting: Multihost developement
+There are three recommended patterns for running MaxText with more than one host.
```
Comment (rwitten) [inline README.md:73]: "three patterns". Only one is recommended, right?

File: getting_started/Run_MaxText_via_xpk.md [hunk 4]
```diff
@@ -76,29 +76,42 @@ after which log out and log back in to the machine.
...
+
+      ```shell
       git clone https://github.com/google/xpk.git
-      cd xpk
 
-      python3 xpk.py workload create \
+      # Make sure you are still in the maxtext github root directory when running this command
+      python3 xpk/xpk.py workload create \
       --cluster ${CLUSTER_NAME} \
-      --docker-image gcr.io/${PROJECT_ID}/${USER}_runner \
```
Comment (rwitten) [inline getting_started/Run_MaxText_via_xpk.md:101]: Beautiful. Feels good, right?

**Review by rwitten — CHANGES_REQUESTED** [review]
Tiny nit. Let's also have @gobbleturk review?

File: getting_started/Run_MaxText_via_xpk.md (hunk 4 above)
Comment (Obliviour) [reply getting_started/Run_MaxText_via_xpk.md:101]: Yes it is amazing. I just can't believe this was supported and I didnt realize.

File: getting_started/Run_MaxText_via_xpk.md [hunk 5]
```diff
@@ -76,29 +76,42 @@ after which log out and log back in to the machine.
...
+      xpk workload create \
+      --cluster ${CLUSTER_NAME} \
+      --base-docker-image maxtext_base_image \
+      --workload ${USER}-first-job \
+      --tpu-type=v5litepod-256 \
+      --num-slices=1  \
+      --command "python3 MaxText/train.py MaxText/configs/base.yml base_output_directory=${BASE_OUTPUT_DIR} dataset_path=${DATASET_PATH} steps=100 per_device_batch_size=1"
+      ```
+
+      __Using [xpk github repo](https://github.com/google/xpk.git)__
```
Comment (gobbleturk) [inline getting_started/Run_MaxText_via_xpk.md:106]: Should we change this documentation for the pip install xpk instead?

**Review by gobbleturk — APPROVED** [review]

**Review by rwitten — APPROVED** [review]

**Pull request merged**